pkgs/tools/misc/overturemaps/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pkgs,
+  setuptools,
+  wheel,
```

```suggestion
 python3Packages,
 fetchFromGitHub,
```
never pass `pkgs` to a package